accounts/views.py

```diff
@@ -1069,3 +1065,33 @@ def pending(request):
     }
     tvars.update(paginate(request, pendings, settings.SOUNDS_PENDING_MODERATION_PER_PAGE))
     return render(request, 'accounts/pending.html', tvars)
+
+
+@csrf_exempt
+def donation_complete(request):
+    params = {'cmd': '_notify-validate'}
+    for key, value in request.POST.items():
+        params[key] =  value
+    if settings.DEBUG:
+        paypal_validation_url = "https://www.sandbox.paypal.com/cgi-bin/webscr"
+    else:
+        paypal_validation_url = "https://www.paypal.com/cgi-bin/webscr"
+    r = requests.post(paypal_validation_url, data=params)
+    if r.text == 'VERIFIED':
+        Donation.objects.get_or_create(transaction_id=params['txn_id'], defaults={
+        'email': params['payer_email'],
+        'display_name': params['custom'],
+        'amount': params['mc_gross'],
+        'currency': params['mc_currency']})
+    return HttpResponse("OK")
+
+
+def donate(request):
+    tvars = {'paypal_email': settings.PAYPAL_EMAIL}
+    return render(request, 'accounts/donate.html', tvars)
+
+
+def donors(request):
+    donations = Donation.objects.all()[:10]
```

Comment:
It might be that donation objects can be anonymous or not anonymous, so the name might appear as anonymous. Is this the case Andrés?
We should probably paginate this. I'm not a big fan of Django's class-based views, but if we simply want to show a paginated list of donors this can probably be done in less than five lines of code using a [ListView](https://docs.djangoproject.com/en/1.10/ref/class-based-views/generic-display/#listview) or something like this.